feature/experimental/bgp/ate_tests/base_bgp_session_parameters/README.md

```diff
@@ -0,0 +1,97 @@
+## RT-1.1: Base BGP Session Parameters
+
+## Summary
+
+    Base BGP session establishment, with required capabilities
+
+## Procedure
+
+    Establish BGP session on ATE and DUT with table-based configuration to cover:
+        ○ Explicitly specified Router ID
+        ○ MD5 password
+        ○ Explicitly specified device-global same local AS number and remote AS number - iBGP
+        ○ Explicitly specified device-global same local AS number and remote AS number - eBGP
+        ○ Explicitly specified per-neighbor local AS number and remote AS number - eBGP
+        ○ Explicitly specified local IP address
+        ○ Explicit holdtime and keepalive interval
+        ○ Explicitly specified retry interval
+    ● Validate session state and capabilities received on ATE.
+    ● Validate session state and capabilities received on DUT using telemetry.
+    ● TODO: Terminate BGP session using NOTIFICATION message and ensure that device telemetry correctly   reports the error codes.
+
+## Config Parameter Coverage
+
+    For prefix:
```

Comment:
Format the "for prefix ... parameters" as shown in this example?

https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/isis/ate_tests/base_adjacencies_test/README.md